### DISCUSSION TITLE
How do you enable GitHub 2FA

### DISCUSSION THREAD

**Discussion by skanda890** [community/community#75939] [Other Feature Feedback, Questions, & Ideas] [answered]
How can you enable GitHub 2FA.

**Comment by goober86** [comment]
It's a good practice to enable 2FA whenever it's available on websites you use regularly to protect your account from unauthorized access.

**Reply by skanda890** [reply]
I asked how to enable it on GitHub.

**Comment by skanda890** [comment, marked as answer]
Enabling two-factor authentication (2FA) on GitHub is a great way to secure your account. Here's how you can do it:

1. Log in to [GitHub](https://github.com/): Go to GitHub and log in to your account.

2. Navigate to Settings: Click on your profile picture in the upper-right corner, then select Settings.

3. Access Security Settings: In the left sidebar, click on Password and authentication under the Access section.

4. Enable 2FA: Under the Two-factor authentication section, click the green button that says Enable two-factor authentication.

5. Choose a 2FA Method: You can choose to set up 2FA using a TOTP (Time-based One-Time Password) app like Google Authenticator or Authy, or via SMS.

6. Follow the Instructions: Follow the on-screen instructions to complete the setup. This usually involves scanning a QR code with your authenticator app or entering a code sent via SMS.

7. Save Recovery Codes: Make sure to save the recovery codes provided by GitHub. These codes can be used to access your account if you lose access to your 2FA method.

By The Internet
